lib/Testable.rb

```diff
@@ -4,8 +4,9 @@ module Testable
 
 
   # functional helpers
-  def test_login
-    session[:user_id] = Fabricate(:user).id
+  def test_login(user = nil)
+    return session[:user_id] = Fabricate(:user).id if user == nil
```

Comment:
I would change this to the much more readable:

```ruby
user ||= Fabricate(:user)
```

The next line will set the id in the session anyway for you and this will make the method much easier to read..